cmd/lakectl/cmd/merge.go

```diff
@@ -41,14 +41,14 @@ var mergeCmd = &cobra.Command{
 	),
 	Run: func(cmd *cobra.Command, args []string) {
 		client := getClient()
-		rightRefURI := uri.Must(uri.Parse(args[0]))
-		leftRefURI := uri.Must(uri.Parse(args[1]))
+		sourceRef := uri.Must(uri.Parse(args[0]))
```

Comment:
Why another terminology? Let's stick with either source/destination or theirs/ours (I like the first one better but that's just me)